Source/Scene/Material.js

```diff
@@ -788,6 +788,11 @@ define([
                         id : uniformId,
                         image : uniformValue
                     });
+                }else if(uniformValue instanceof HTMLImageElement){
+                    material._loadedImages.push({
+                        id : uniformId,
+                        image : uniformValue
```

Comment:
Since this block is a duplicate of the `HTMLCanvasElement` case, can we just add and additional or for `HTMLImageElement`?

Reply:
@polluxll Please address this comment when you have a minute.  Thanks!